### ISSUE TITLE
Should "Positive charges do  not move." be in the scene summary item for induced charge?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#377]
@terracoda I noticed in the design document that induced charge is described like this:

> Negative charges in {{wall}} move away from {{yellow}} balloon {{a little}}, from {{green balloon}} {{a lot}}.

None of the examples have the phrase "Positive charges do not move", which is appended in other descriptions for induced charge. I will remove it since it is not in the design doc, but wanted to make sure this was intended.

**Comment by terracoda** [comment]
@jessegreenberg, I think this is intentional. I am reviewing the examples to be sure.

**Comment by terracoda** [comment]
@jessegreenberg, actually, I'm a little on the fence about whether "Positive charges do not move." should be removed from Scene Summary's induced charge description.

The phrase is included when there is an induced charge effect most prominently during 
- interactive rubbing up and down the wall, and 

it also is included when there is an induced charge effect (i.e., a charged balloon is at the wall):
- in the Wall's dynamic description 
- in the Grab alert
- upon first contact with wall during the dragging interaction

The only place where I explicitly removed it was in the [+ Induced charged (simple)] pattern used in the Release alerts for releases that end at the wall. 

Perhaps for consistency, we should leave it in the Scene Summary. Is it too much effort to put it back?

**Comment by jessegreenberg** [comment]
Not at all @terracoda, happy to add it back.

**Comment by terracoda** [comment]
@jessegreenberg, ok I will update the Scene Summary descriptions in the Two-balloon design document. Sorry for the confusion - oh so many descriptions. Thanks for your questions!

**Comment by jessegreenberg** [comment]
"Positive charges do not move." has been added to induced charge item in the scene summary. I tested the various cases:
![capture](https://user-images.githubusercontent.com/6396244/38218349-ae9e4e60-368e-11e8-9894-66d60f3256a6.PNG)
![capture2](https://user-images.githubusercontent.com/6396244/38218351-af93bb7a-368e-11e8-8967-a2a7883c4f84.PNG)
![capture3](https://user-images.githubusercontent.com/6396244/38218354-b0dbdcd8-368e-11e8-9250-28bff391905b.PNG)

Closing.
